src/library_egl.js

```diff
@@ -277,7 +279,7 @@ var LibraryEGL = {
 
   // EGLAPI EGLSurface EGLAPIENTRY eglCreateWindowSurface(EGLDisplay dpy, EGLConfig config, EGLNativeWindowType win, const EGLint *attrib_list);
   eglCreateWindowSurface__proxy: 'sync',
-  eglCreateWindowSurface__sig: 'iiiii',
+  eglCreateWindowSurface__sig: 'ppppp',
```

Comment:
I'm working a set of changes that should make these automatic: #18979.    Hopefully the final part will simply remove these lines!

I'm not sure when it will land but hopefully most of it should land today.

Reply:
Nice, that would really simplify this pull request significantly.